leap.json

```diff
@@ -26,7 +26,7 @@
          "expected": true
       },
       {
-         "description": "Y2K",
+         "description": "y2k",
```

Comment:
There's a bit of a catch here. If I don't change this, the Robucop check fails as the Y2K transfers through to the method name of the test, which becomes test_Y2K. 
The current logic in the test generators appears to expect the case in the JSON files to be lower case, and in Ruby, intersperses with a "_" to make valid names. This fails in the case of Y2K.
It feels like there is a design decision to be made regarding what the generators can expect, and for them to code it up according. The current implicit design decision in the Ruby generators would appear to be to expect lower case descriptions, and create test method names accordingly.
By me changing the case to lower, the current Ruby generator pattern works. Looking at the Go generator, it appears as though it does not use the description for generating a test method name, so no issue.